### PR TITLE
Doc update2

### PR DESCRIPTION
**Pull request by yuokamoto**
add documents
- external device
- payload handling area
- character

https://rapyutasimulationplugins.readthedocs.io/en/doc_update2/external_devices.html

### CODE REVIEW

File: docs/source/external_devices/payload_handling_area.rst [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+          [
+            {
+              "name": "teleport_area1",
+              "transform": …,
+              "teleport_mode": …,
+              …
+            },
+            {
+              "name": "teleport_area1",
```
Comment (tetorea) [inline docs/source/external_devices/payload_handling_area.rst]: `name` doesn't need to be unique? its value is the same as the block above

File: docs/source/external_devices/payload_handling_area.rst [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+     - `example_interfaces/msg/Int32 <https://docs.ros2.org/foxy/api/example_interfaces/msg/Int32.html>`_
+     - Sets the mode: 0 for manual, 1 for auto.
+   * - /set_duration
+     - `example_interfaces/msg/Float32 <https://docs.ros2.org/foxy/api/example_interfaces/msg/Float32.html>`_
+     - Sets the duration for auto-triggering actions.
+   * - /trigger
+     - `example_interfaces/msg/Empty <https://docs.ros2.org/foxy/api/example_interfaces/msg/Empty.html>`_
+     - Manually triggers an action.
+
+Sink Area
```
Comment (tetorea) [inline docs/source/external_devices/payload_handling_area.rst:75]: nit: for me, "sink" has the image of element going down but it's not obvious the element gets deleted

File: docs/source/external_devices/payload_handling_area.rst [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+     - `example_interfaces/msg/Int32 <https://docs.ros2.org/foxy/api/example_interfaces/msg/Int32.html>`_
+     - Sets the spawn mode:  
+
+       0. Spawns one actor at a time,  
+       1. Spawns all actors together.
+
+Teleport Area and Manager
+-------------------------
+
+`BP_TeleportArea` is a child class of `BP_SinkArea`, and `BP_TeleportAreaManager` is a child class of `BP_PayloadHandlerBase`. `BP_TeleportAreaManager` holds a list of `BP_TeleportArea` instances, and manually triggering its action will trigger all teleport areas in the list.
```
Comment (tetorea) [inline docs/source/external_devices/payload_handling_area.rst:161]: BP_TeleportArea being a child of BP_SinkArea, does it means the object is deleted then spawned?

**Review by tetorea — APPROVED** [review]

File: docs/source/external_devices/payload_handling_area.rst (hunk 3 above)
Comment (yuokamoto) [reply docs/source/external_devices/payload_handling_area.rst:161]: no. just use same logic to trigger event

File: docs/source/external_devices/payload_handling_area.rst (hunk 2 above)
Comment (yuokamoto) [reply docs/source/external_devices/payload_handling_area.rst:75]: - In that case, DeleteSpot make sense?
- Does SourceSpot make sense?

**Comment by yuokamoto** [comment]
temporary merge this pr. will rename and update documentation if ncesary later

**Pull request merged**